github/ci/prow-deploy/files/jobs/kubevirt/libguestfs-appliance/libguestfs-appliance-presubmits.yaml

```diff
@@ -0,0 +1,29 @@
+presubmits:
+  kubevirt/libguestfs-appliance:
+  - name: check-build-libguestfs-appliance
```

Comment:
the recommended naming pattern would be `pull-libgiuestfs-appliance-build`

Reply:
ok, I changed it.